modules/oc-adm-by-example-content.adoc

```diff
@@ -340,31 +340,72 @@ Create a new project
 
 
 
+== oc adm node-image create
+Create an ISO image for booting the nodes to be added to the target cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Create the ISO image and download it in the current folder
+  oc adm node-image create
```

Comment:
```./oc adm node-image create
error: stat nodes-config.yaml: no such file or directory```
command failed.

Reply:
@zhouying7780 if I look at the help for `oc adm node-image create`, it does say this:

> A nodes-config.yaml config file must be created to provide the required initial configuration for the selected nodes.

So I think this means that this file must exist (either in the current folder for this command, or in the /tmp/assets/ folder for the next command) for these commands to work.

Reply:
That's correct. If the `--dir` option is not configured (and no single node flag is used), by default the asset is equal to the current working directory